recipes/vulkan-headers/all/conanfile.py

```diff
@@ -14,6 +14,7 @@ class VulkanHeadersConan(ConanFile):
     homepage = "https://github.com/KhronosGroup/Vulkan-Headers"
     url = "https://github.com/conan-io/conan-center-index"
     package_type = "header-library"
+    package_id_embed_mode = "patch_mode"
```

Comment:
I don't see any reference to `package_id_embed_mode` attribute in conan v2 documentation, and even less possible values and their meaning.

Reply:
@SpaceIm it's documented now: https://docs.conan.io/2/reference/conanfile/attributes.html#package-id-embed-non-embed-unknown-mode